### PR TITLE
[MODULAR] [ATOMIZED] Adds Free Trade Union ERT, lays out groundwork for more faction ERTs in the near future [PART 1]

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

This is the first part of a two-part PR that adds an FTU focused ERT into the game.

## Why It's Good For The Game

We have Armadyne Corpos, I feel like that's justification for us to add faction-specific ERTs for every kind of threat departments may face. The FTU ERT, for example, is equipped to deal with dangerous cargo and to repel pirate boarding parties.

This first part contains all the code and sounds for their equipment! Part two will have their clothing, loadout, and datum so that they're ready for deployment.

## Changelog
:cl:
add: Added R-37 Pulse Rifle, R-40 PMG, PDH-6G and Ignis plasma sword (All with custom sprites and sounds!)
add: Added 6.5mm XJP, 7.2mm XJP and 12.7x35mm SAP-HE bullet types, including their own tracers to easily distinguish them.
soundadd: Ballistic pulse rifle, plasma sword and pulse magnum sounds.
imageadd: Added all necessary icons for the FTU ERT
code: Laid out groundwork for modular faction ERTs in their own unique folder.
/:cl:

CREDITS AND CONTENT ORIGINS:
R37 pulse rifle is an extremely heavily changed sprite originated from Halo station. It was upscaled, recolored, and had its shape almost drastically changed.
R40 machine gun's original sprite comes from CEV Eris.

Plasma sword and Magnum sounds from Bungie's Halo 3
Pulse rifle sounds from SEGA's Aliens: Colonial Marines

### CODE REVIEW

**Comment by Draganfrukts** [comment]
why the fuck does it have a code improvement tag

**Comment by Draganfrukts** [comment]
oh god critical issue

File: modular_skyrat/modules/ERT_Factions/FTU/sound/placeholder.dm [hunk 1]
```diff
@@ -0,0 +1 @@
+how he stand on 10 perc?
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/ERT_Factions/FTU/sound/placeholder.dm]: removeeee

Comment (Draganfrukts) [reply modular_skyrat/modules/ERT_Factions/FTU/sound/placeholder.dm]: There we go I'll just remove all the placeholder files

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by Oscari-pog** [comment]
I like the diversity, but creating a ERT specifically to steamroll pirates is oof

**Comment by Draganfrukts** [comment]
> I like the diversity, but creating a ERT specifically to steamroll pirates is oof

They're there for stuff that affects cargo, and should be treated like armadyne because their gear is arguably equally if not more powerful. You shouldn't be calling it over just 3 pirates

**Pull request merged**